Empty Lines Bug

```
        if isinstance(formatted_obj, ItemFormat):
>           max_len = max(map(BaseFormat.len, text.splitlines()))
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
E           ValueError: max() iterable argument is empty

../../miniconda3/envs/arc/lib/python3.13/site-packages/debug/_format.py:499: ValueError
```

--- a/debug/_code.py
+++ b/debug/_code.py
@@ -51,9 +51,9 @@ def get_source_segments(source: str) -> None | Iterable[str]:
 def get_source(frame: types.FrameType) -> None | str:
     try:
         lines, offset = inspect.getsourcelines(frame)
-        offset = max(offset, 1)
     except OSError:
         return None
+    offset = max(offset, 1)
     traceback = inspect.getframeinfo(frame, context=0)
     positions = traceback.positions
     if positions is None:

--- a/debug/_format.py
+++ b/debug/_format.py
@@ -495,7 +495,7 @@ class Formatter:
         formatted_obj = BaseFormat._from(obj, set())
         text = formatted_obj._format(initial_width, highlight=True, config=self._config)
         terminal_width = self._config._terminal_width
-        if isinstance(formatted_obj, ItemFormat):
+        if isinstance(formatted_obj, ItemFormat) and text:
             max_len = max(map(BaseFormat.len, text.splitlines()))
             if terminal_width is not None and max_len + initial_width > terminal_width:
                 text = "\n" + text

--- a/pyproject.toml
+++ b/pyproject.toml
@@ -1,6 +1,6 @@
 [project]
 name = "debug"
-version = "1.5.0"
+version = "1.5.1"
 dynamic = ["dependencies"]
 
 [build-system]

--- a/test_suite.py
+++ b/test_suite.py
@@ -494,6 +494,11 @@ class DataclassCustomRepr:
         return "DataclassCustomRepr!"
 
 
+class EmptyRepr:
+    def __repr__(self) -> str:
+        return ""
+
+
 @pytest.mark.parametrize(
     "obj, width, expected",
     [
@@ -1626,6 +1631,8 @@ class DataclassCustomRepr:
             None,
             "freezedict()",
         ),
+        (EmptyRepr(), None, ""),
+        (EmptyRepr(), 1, ""),
     ],
 )
 def test_format(obj: Any, width: int | None, expected: list | str) -> None:

